internal/integration/integration.go

```diff
@@ -48,31 +49,48 @@ func RunFlex(runMode FlexRunMode) {
 
 	switch runMode {
 	case FlexModeLambda:
-		addConfigsFromPath("/var/task/pkg/flexConfigs/", &configs)
+		errors := addConfigsFromPath("/var/task/pkg/flexConfigs/", &configs)
+		if len(errors) > 0 {
+			return fmt.Errorf("flex: failed to read some configuration files. please review them before continuing")
```

Comment:
I would use a comma instead of period, since we are not ending the sentence

Reply:
also should this not also be logrus.WithError()?

Reply:
this is one of those cases where I was not sure if we should stop processing if we encounter an "unrecoverable" in one of the files we're trying to load.
If we continue we (the user) may not notice it and question themselves why one of his configurations is not reporting.
If you're feeling is that we should continue lmk and I'll change the behaviour, but I do think having a "healthy" config folder is better than having some random file failing